scripts/apps/monitoring/directives/MonitoringGroup.ts

```diff
@@ -520,7 +535,8 @@ export function MonitoringGroup(cards, api, authoringWorkspace, $timeout, superd
                         }
 
                         // custom sort for group (if it exists)
-                        if (scope.customSortOptionActive && scope.customSortOptionActive.field !== DEFAULT_SORT_FIELD) {
+                        if (scope.customSortOptionActive &&
+                            scope.customSortOptionActive.field !== USE_DEFAULT_SORT_FIELD) {
```

Comment:
what about having entire `customSortOptionActive` equal to `null` when a custom sort is not set? Having it set and having a field equal `null` is less clear.